doc/src/hugo/content/concepts/processor.md

```diff
@@ -4,9 +4,9 @@ draft: false
 ---
 
 A processor is a component of any [vital definition]({{< relref "vital" >}}). 
-Processors perform some transformation, typically, of the data flowing in from
```

Comment:
I think the word "typically" is useless here. The only transformation processors should do is of their input from inlets. So the typical case is the always cas too. Could you just remove "typically" from that sentence, please?

Reply:
Same thing with "potentially diffferent" below.  If there was no transformation, there would be no need to have a processor at all.

Reply:
Should it be "produces some different data" or just "produces some data"?

Reply:
_A processor performs some transformation of the data flowing in from its [inlet](s) and produces different data to its [outlet](s)._

Reply:
fixed!